### DISCUSSION TITLE
Github appliance

### DISCUSSION THREAD

**Discussion by rnadupal** [community/community#64557] [Enterprise] [answered]
Hello,

I am new to GH Enterprise and I have deployed the appliance in the ESXI environment. I have changed the hostname from the UI twice, however after rebooting the host, services are not loading. Seeing the following when I run systemctl list-jobs. in the setup screen it stuck at reloading system services. Any suggestions on how to fix this?

![image](https://github.com/community/community/assets/142362564/69f67b34-9c10-4463-8774-37b97bd9d259)

JOB UNIT                                 TYPE  STATE
320 ghe-reconfigure.service              start running
221 graphical.target                     start waiting
222 multi-user.target                    start waiting
546 nomad-jobs.service                   start running
317 systemd-update-utmp-runlevel.service start waiting

**Comment by queenofcorgis** [comment]
Hi there - I'm going to share this with my team and see if we can get some answers!

**Comment by rnadupal** [comment]
Found the issue.

**Reply by emeraldo** [reply]
Sooooo can you share the solution? This is literally the only spot on the Internet that talks about this issue and I too need the resolution.

**Reply by rnadupal** [reply, marked as answer]
It turned out to be a resource issue on the VMs. Since it was a fresh install, I started off with low CPU and memory. Because of that everything was super slow and taking ages to commit the config.  After matching the appliance VM size to the below-recommended configuration, everything worked fine. 

https://docs.github.com/en/enterprise-server@3.8/admin/installation/setting-up-a-github-enterprise-server-instance/installing-github-enterprise-server-on-vmware

**Reply by queenofcorgis** [reply]
We’re glad you could get this solved and we want to make sure others can do the same so we’ve reopened this discussion for better discoverability for our members.
